app/models/refusal_advice.rb

```diff
@@ -81,6 +79,12 @@ def ==(other)
 
   private
 
+  def build_from_data(key)
+    Array(data.dig(legislation.to_sym, key)).map do |value|
+      "#{ self.class }::#{ key.to_s.classify }".constantize.new(value)
```

Comment:
Not sure why Rubocop didn't catch this:
```suggestion
      "#{self.class}::#{key.to_s.classify}".constantize.new(value)
```

Reply:
> Not sure why Rubocop didn't catch this

This [check is disabled](https://github.com/mysociety/alaveteli/blob/0f831a00b134c5a244859391ff2031d11de339a0/.ruby-style.yml#L324-L325) and has been [since it was added](https://github.com/mysociety/alaveteli/commit/a45df67b30dbc48accd6391d7067280be3c4a994).

I can't quickly find anything where this decision has been documented, but Alaveteli has always implicitly used [`EnforcedStyle: space`](https://docs.rubocop.org/rubocop/1.0/cops_layout.html#enforcedstyle-space-4).